### PR TITLE
Update messages.en_GB.yml

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: app/resources/translations/en_GB/messages.en_GB.yml [hunk 1]
```diff
@@ -77,11 +72,21 @@
...
+"Related content": "Related content"
 "Repeater": "Repeater"
+"Search": "Search"
+"Search results for <b>%search%</b>.": "Search results for <b>%search%</b>."
+"Search …": "Search …"
 "Select a page": "Select a page"
 "Select zero or more pages": "Select zero or more pages"
 "Selectentry": "Select entry"
 "Selectfield": "Select field"
+"Seo": "Seo"
```
Comment (GawainLynch) [inline app/resources/translations/en_GB/messages.en_GB.yml]: s/Seo/SEO/

…and, um… we don't use that string, do we?

Comment (bobdenotter) [inline app/resources/translations/en_GB/messages.en_GB.yml]: Ah, that's #4161 rearing it's ugly head again. I'll remove that line!

**Comment by bobdenotter** [comment]
*bump* :-)

**Comment by GawainLynch** [comment]
Ooops :laughing:

**Pull request merged**